python/python/geoindex_rs/tests/test_buffers.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+from .. import RTree
+
+def generate_random_boxes():
+    random_points = np.random.random_sample((100, 2))
+    boxes = np.concatenate([random_points, random_points], axis=1)
+    return boxes
+
+def test_buffer_protocol():
+    boxes = generate_random_boxes()
+    initial = RTree.from_interleaved(boxes)
+    # construct a memoryview transparently
+    view = memoryview(initial)
+    assert initial.num_bytes == view.nbytes
```

Comment:
Nit: I usually set up pre-commit to enforce a newline at the end of file, though I haven't done that yet here.